lib/membrane/rtp/jitter_buffer/buffer_store.ex

```diff
@@ -72,10 +72,12 @@ defmodule Membrane.RTP.JitterBuffer.BufferStore do
          buffer,
          seq_num
        ) do
+    base_index = rem(prev_index, @seq_number_limit)
```

Comment:
If you do the reminder with `@seq_number_limit` then it is no longer an index. Index is a counter that cummulates sequence numbers with their respective rollovers (while sequence number is 16 bits then the index can be much larger containing multiple sequence number maximum valu). In this case just name it `prev_seq_num`.